main/options/options.cc

```diff
@@ -673,11 +671,9 @@ void readOptions(Options &opts,
                                   opts.inputFileNames.end());
 
         bool enableAllLSPFeatures = raw["enable-all-experimental-lsp-features"].as<bool>();
-        bool enableBetaLSPFeatures = enableAllLSPFeatures || raw["enable-all-beta-lsp-features"].as<bool>();
+        // bool enableBetaLSPFeatures = enableAllLSPFeatures || raw["enable-all-beta-lsp-features"].as<bool>();
```

Comment:
Do you want to put a description here saying what's up?

It's usually a code smell to leave code checked in but commented out, though I'm happy to have a comment here explaining why we didn't completely delete this.

Reply:
Good point.  (Lint nudged me to comment it out, but I didn't want to delete it because I *believe* that Sushain is about to use it next.)   I'll address your point (hopefully!) by defining it in the headers file so that it's no longer commented-out code.